distribution/lib/Standard/Base/0.0.0-dev/src/Data/Time/Date.enso

```diff
@@ -826,6 +826,11 @@ type Date
     format self format:Date_Time_Formatter=Date_Time_Formatter.iso_date =
         format.format_date self
 
+    ## PRIVATE
+       Convert to a Enso code representation of this Date.
+    pretty : Text
+    pretty self = "(Date.new " + self.year.to_text + " " + self.month.to_text + " " + self.day.to_text + ")"
```

Comment:
Eh! Another way to persist a value.

Wouldn't it be better to have `type Code` and then conversions to instances of the `Code` type? Because right now you get a code, but you may not have the right imports! `Code` could carry both - the imports and the code to construct the instance.

Reply:
This is matching our current `pretty` methods which work for *some* types, we can happily review and work out what we should do longer term.